src/Serilog.Sinks.Raygun/Sinks/Raygun/RaygunSink.cs

```diff
@@ -107,8 +107,8 @@ public void Emit(LogEvent logEvent)
                 ? RaygunErrorMessageBuilder.Build(logEvent.Exception)
                 : new RaygunErrorMessage()
                 {
-                    ClassName = logEvent.MessageTemplate.Text,
-                    Message = logEvent.MessageTemplate.Text,     
+                    ClassName = logEvent.RenderMessage(_formatProvider),
+                    Message = logEvent.RenderMessage(_formatProvider),     
```

Comment:
It seems like rendering `Message` fits with the `RaygunErrorMessage` ~= `Exception` mapping, but if `ClassName` is analogous to the exception's type, then `MessageTemplate.Text` might still be a good fit, there. The comment a few lines above points out that the template is useful for grouping - will this still work if only `ClassName` is a template? (Seems like it should 👍)

Reply:
I'm not sure how anybody else is using the message templates, we use Serilog with sinks to Seq, and Console as well as Raygun (only for production). 
This change aligns the Raygun sink with how it was implemented for Seq and Console

I had a look at the Serilog documentation and the way o log the message would be with a message template, which then replaces the values (https://github.com/serilog/serilog/wiki/Formatting-Output). 

As to whether existing functionality would break, I have no idea, 
For us for example, we would need to fine tune our Raygun groupings to look at only the first part of the message and group on that, which is how we have set up our message template.
Raygun seems to be using the Classname for the grouping.

Expected behavior being:
Supply message template, with properties to populate the message.
eg: HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms
replaced to: "HTTP "POST" "/home/calculations" responded 500 in 3413.3153 ms

Actual behavior:
Message template displayed as it was written (No parameters replaced).
![image](https://user-images.githubusercontent.com/36261705/65251049-ec226d80-daf6-11e9-9a42-aa7e8aa7be61.png)

Reply:
I am not sure I follow completely - I agree regarding `Message`, which also seems to line up with your screenshot. Changing `ClassName` to also be the rendered message still seems strange, though. (I've also never personally used Raygun, so it's hard for me to judge how much this will impact existing users of the sink.)

Reply:
@QuantumNightmare any thoughts on this?

Reply:
Thanks for the ping. I also agree that the Message should include the actual values, not just the template.

By default, if the client that sent the crash report is unknown (which is currently the case for the Raygun sink), then Raygun performs grouping based on just the Message, not the ClassName. So even just changing the Message is going to cause a change in grouping behaviour. Arguably, it's more of a bug fix than a breaking change because the outcome is improved messages. However the provided example includes a very variable value - elapsed time which could mean 1 new group for every log instance.

As for ClassName, for the lack of anything else, I think the raw template (as it currently is) makes the most sense. With the absence of any stack traces in this kind of logged message, the ClassName as the raw template could be a good option for default exception grouping. This is something we'd need to wire up on Raygun's side though which I can find some time to implement. Another option might be to give this type of log message a StackTrace. Normally this comes from an Exception, but with the lack of an Exception, it should still be possible to use the current Stack and attach that to the Raygun error message. This would be nice if you wanted to group log messages based on where they occurred.

Summary:
* Message is updated to format values into the template.
* ClassName is left as it currently is - the raw log message template.
* Ideally add a Stack trace to the Raygun report for these kind of exceptionless logs.
* Here at Raygun, we can default exceptions from this sink to group by stack trace (if that gets added as part of this change) or otherwise ClassName (which would result in no change to the current grouping). As long as this part is put in place, I feel this would count as a bug fix / improvement rather than a breaking change.

Reply:
Just noting that improving the grouping logic for the Serilog Raygun Sink on our end is something that we do want to do anyway. We may be looking into this soon.
@intermediateza Would you like to make the other changes to this PR? Change ClassName back to being the template, and adding the current execution Stack Trace. If not, I might spin up an alternative PR to do this, or let me know if you have any questions about what's been mentioned here.